### ISSUE TITLE
Source Code links broken in Ruby docs

### ISSUE THREAD

**Issue by omaray** [GoogleCloudPlatform/google-cloud-ruby#908]
It seems like all the `Source Code` links in the Ruby docs are broken at the moment.

As an example, if you navigate to the [root of the docs](https://googlecloudplatform.github.io/google-cloud-ruby/#/docs/google-cloud/v0.20.0/google/cloud) and look for the first `Source Code` link, it won't work.

And this seems to be the case for all of them:
![image](https://cloud.githubusercontent.com/assets/6609430/18619329/68c02246-7dae-11e6-93e1-df29c077ff86.png)

**Comment by quartzmo** [comment]
Thanks @omaray. I think fixing this will require an update to our JSON docs generation. I'll look into it.

**Comment by quartzmo** [comment]
GoogleCloudPlatform/gcloud-common#188 added support to the site for git tags containing both package name and version number, solving one cause of this issue.

I've fixed the links in all individual packages manually (and will follow up with automation), but the links for the "umbrella" package will require a change to our JSON docs generation.

Example of a current (broken) path:

https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/google-cloud/v0.20.0/google-cloud/lib/google-cloud-bigquery.rb#L65

And the correct path:

https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/google-cloud/v0.20.0/google-cloud-bigquery/lib/google-cloud-bigquery.rb#L65

At the same time that I make this fix, I will probably produce the automation for the manual updates I made earlier.
